src/core/storage/Cookie.ts

```diff
@@ -1,8 +1,31 @@
+import * as R from 'ramda';
+
 const __1day = 86400 * 1000;
 const __20years = 86400 * 1000 * 365 * 20;
 
 export default class CookieStorage {
+    // From https://github.com/Modernizr/Modernizr/blob/f4d3aa0b3c9eeb7338e8d89ed77929a8e969c502/feature-detects/cookies.js#L1
+    // try..catch because some in situations `document.cookie` is exposed but throws a
+    // SecurityError if you try to access it; e.g. documents created from data URIs
+    // or in sandboxed iframes (depending on flags/context)
+    public static enabled = R.once((): boolean => {
+        try {
+            // Create cookie
+            document.cookie = 'cookietest=1';
+            let ret = document.cookie.indexOf('cookietest=') !== -1;
+            // Delete cookie
+            document.cookie = 'cookietest=1; expires=Thu, 01-Jan-1970 00:00:01 GMT';
+            return ret;
+        } catch (e) {
+            return false;
+        }
+    });
```

Comment:
@josegonzalez As I'm nearing a release and would want to get this in, took the liberty of updating this code so that the `CookieStorage.enbled` runs only once.